### PR TITLE
perf(solution): can only add sso for app service hosted bot

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts [hunk 1]
```diff
@@ -852,6 +859,21 @@ export function canAddSso(
     );
   }
 
+  // Can only add sso for bot with host type == 'app-service'
```
Comment (a1exwang) [inline packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts:862]: Better to clarify this comment. `hostType === 'azure-functions'` is func hosted bot. `hostType === 'app-service'` or `undefined` is app service hosted bot. `undefined` is for legacy projects. Do you support legacy bot projects (for example bot project scaffolded before GA release)?

Comment (KennethBWSong) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts:862]: @a1exwang Will migration add host type for previous projects?

Comment (a1exwang) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts:862]: No. host type exists only for newly created projects.

**Review by SLdragon — DISMISSED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts (hunk 1 above)
Comment (KennethBWSong) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts:862]: OK, then I will just update the comment. Thx！

**Review by 1yefuwang1 — APPROVED** [review]

**Pull request merged**